packages/manager/src/features/CloudPulse/shared/CloudPulseRegionSelect.tsx

```diff
@@ -27,6 +38,13 @@ export const CloudPulseRegionSelect = React.memo(
       selectedDashboard,
     } = props;
 
+    const serviceType: string | undefined = selectedDashboard?.service_type;
+    const capability = React.useMemo<Capabilities | undefined>(() => {
+      return serviceType
+        ? FILTER_CONFIG.get(serviceType)?.capability
+        : undefined;
+    }, [serviceType]);
```

Comment:
```suggestion
    const capability =  serviceType ? FILTER_CONFIG.get(serviceType)?.capability : undefined;
    
```

no need to put it inside memo, anyways it is a constant time operation. Only put things inside memo when you need some processing. Instead directly calculate the value. Avoid using hooks unless it is very necessary